sadcowpng/sharedspaces/models.py

```diff
@@ -1,3 +1,52 @@
 from django.db import models
+from .forms import Noise_Level_Choices
 
-# Create your models here.
+
+# All information that will be sent to space
+class Space(models.Model):
+    space_name = models.CharField(max_length=500)
+    space_description = models.CharField(max_length=1000)
+    space_max_capacity = models.IntegerField()
+    space_noise_level_allowed = models.IntegerField()
+    space_noise_level = models.IntegerField()
+    space_wifi = models.BooleanField()
+    space_restrooms = models.BooleanField()
+    space_food_drink = models.BooleanField()
+
+    # space_open = models.BooleanField()
+
+    # Might want to create a string method for each data type
```

Comment:
Again, merged comments should reflect a decision that was taken, not reflect a consideration that you were making while developing.

Reply:
Fixed